functions/Remove-SqlOrphanUser.ps1

```diff
@@ -89,19 +92,19 @@ Will remove from all databases the user OrphanUser EVEN if exists their matching
 	)
 
     DynamicParam { if ($SqlServer) { return Get-ParamSqlDatabases -SqlServer $SqlServer -SqlCredential $SqlCredential } }
-	
+
 	BEGIN
 	{
         Write-Output "Attempting to connect to Sql Server.."
 		$server = Connect-SqlServer -SqlServer $SqlServer -SqlCredential $SqlCredential
 	}
-	
+
 	PROCESS
```

Comment:
multiple SqlInstances are allowed but there seems to be no foreach in PROCESS{} ?
This way processing multiple instances can only be processed when piped in with | , but not when given as parameter values with -SqlInstance.
Since connecting to $sqlserver is done in BEGIN{}, that also will only be executed once and not for every instance.
I believe the function will fail to process multiple instances anyway. Has this been tested?